pkg/controller/config/config_controller_test.go

```diff
@@ -469,100 +470,113 @@ func TestConfig_CacheContents(t *testing.T) {
 	}, 10*time.Second).Should(gomega.BeZero(), "waiting for cache to empty")
 }
 
-type opaKey struct {
-	gvk schema.GroupVersionKind
-	key string
-}
-
-// fakeOpa is an OpaDataClient for testing.
-type fakeOpa struct {
-	mu   gosync.Mutex
-	data map[opaKey]interface{}
-}
-
-// keyFor returns an opaKey for the provided resource.
-// Returns error if the resource is not a runtime.Object w/ metadata.
-func (f *fakeOpa) keyFor(obj interface{}) (opaKey, error) {
-	o, ok := obj.(runtime.Object)
-	if !ok {
-		return opaKey{}, fmt.Errorf("expected runtime.Object, got: %T", obj)
-	}
-	gvk := o.GetObjectKind().GroupVersionKind()
-	k, err := cache.MetaNamespaceKeyFunc(obj)
-	if err != nil {
-		return opaKey{}, fmt.Errorf("extracting key: %v", err)
+func TestConfig_Retries(t *testing.T) {
```

Comment:
Thank you for thinking through how to test this

Reply:
Sure thing. I wanted to add a test for the other retry (when wiping fails) - but didn't have enough time to follow through on it.